contrib/babelfishpg_tsql/runtime/functions.c

```diff
@@ -3839,17 +3843,20 @@ bbf_pivot(PG_FUNCTION_ARGS)
 	oldcontext = MemoryContextSwitchTo(tsql_outmost_context);
```

Comment:
Inside the pg try we didn't copy or do any memory allocate, why we switch to outmost context at here ?

Reply:
Removed PG_TRY and, MemoryContextSwitch code.